composer.json

```diff
@@ -5,6 +5,7 @@
     "description": "GiveWP - a WordPress Donation Plugin and Fundraising Platform",
     "homepage": "https://givewp.com/",
     "require": {
+        "php": "^7.2 || ~8.0.0",
```

Comment:
We do already have a composer `platform` config set to php 7.2

Reply:
Okay. What should I do now?

Reply:
If you agree this addition is not necessary since we already have the platform config, we can just close the PR

Reply:
All right!